pkg/controllers/user/clusterauthtoken/setting.go

```diff
@@ -0,0 +1,70 @@
+package clusterauthtoken
+
+import (
+	managementv3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const settingName = "auth-provider-refresh-debounce-seconds"
```

Comment:
This could be moved to common and exported then rancher-kube-api-auth can take advantage of it.